src/cli/self_update/windows.rs

```diff
@@ -929,62 +876,48 @@ mod tests {
 
     #[test]
     fn windows_path_regkey_type() {
-        // per issue #261, setting PATH should use REG_EXPAND_SZ.
```

Comment:
The actual semantics of `_apply_new_path()` and that of this test have been changed from saving a `REG_EXPAND_SZ` to `REG_SZ`, which looks like a clear regression to me (cc #261).

Reply:
I briefly browsed the source code of `windows-registry` and found that it puts `REG_SZ` and `REG_EXPAND_SZ` together to handle strings.

Reply:
> I briefly browsed the source code of `windows-registry` and found that it puts `REG_SZ` and `REG_EXPAND_SZ` together to handle strings.

@InfyniteHeap I see. However this is not the reason not to "do the right thing" as some users might be affected with this change. You might consider calling raw APIs if necessary.

Reply:
cc https://github.com/microsoft/windows-rs/issues/3148